tonic_validate/metrics/answer_match_metric.py

```diff
@@ -0,0 +1,21 @@
+import logging
+
+from tonic_validate.classes.llm_response import LLMResponse
+from tonic_validate.metrics.binary_metric import BinaryMetric
+from tonic_validate.services.openai_service import OpenAIService
+
+logger = logging.getLogger()
+
+
+class AnswerMatchMetric(BinaryMetric):
+    def __init__(self, name: str, answer: str, case_sensitive: bool = False):
+        super().__init__(name, self.metric_callback)
+        self.text = answer
```

Comment:
Nit: this should be `self.answer = answer`.

Reply:
Whoops, I think these mistakes just got carried in from me copy pasting stuff